Services/UI/OJS.Services.Ui.Models/Submissions/SubmissionDetailsServiceModel.cs

```diff
@@ -79,6 +81,8 @@ public void RegisterMappings(IProfileExpression configuration)
                         : s.ContentAsString))
                 .ForMember(d => d.IsOfficial, opt => opt.MapFrom(s =>
                     s.Participant!.IsOfficial))
+                .ForMember(d => d.MemoryLimit, opt => opt.MapFrom(s =>
+                    s.Problem!.MemoryLimit))
```

Comment:
Should be removed for now. The Memory limit logic is about to be updated and this will no longer be valid.

Reply:
done